internal/models/bizmodels/ordermodel/ordermodel.go

```diff
@@ -0,0 +1,93 @@
+package ordermodel
+
+import (
+	"time"
+
+	"github.com/dmitrovia/gophermart/internal/models/bizmodels/usermodel"
+)
+
+const OrderStatusNew string = "NEW"
```

Comment:
You can create a new type, for example `OrderStatusType`.
```suggestion
const New OrderStatusType = "NEW"
```